tests/python_package_test/test_dask.py

```diff
@@ -90,6 +91,57 @@ def test_classifier(output, centers, client, listen_port):
     assert_eq(y, p2)
 
 
+@pytest.mark.parametrize('output', data_output)
+@pytest.mark.parametrize('centers', data_centers)
+def test_classifier_pred_contrib(output, centers, client, listen_port):
+    X, y, w, dX, dy, dw = _create_data('classification', output=output, centers=centers)
+
+    dask_classifier = dlgbm.DaskLGBMClassifier(
+        time_out=5,
+        local_listen_port=listen_port,
+        tree_learner='data'
+    )
+    dask_classifier = dask_classifier.fit(dX, dy, sample_weight=dw, client=client)
+    preds_with_contrib = dask_classifier.predict(dX, raw_score=True, pred_contrib=True).compute()
+
+    local_classifier = lightgbm.LGBMClassifier()
+    local_classifier.fit(X, y, sample_weight=w)
+    local_preds_with_contrib = local_classifier.predict(X, raw_score=True, pred_contrib=True)
```

Comment:
Why do we need `raw_score=True` here? I believe `raw_score` and `pred_contrib` are mutually exclusive. Seems it is just luck that `if pred_contrib` is the last one check and this test is passed.
https://github.com/microsoft/LightGBM/blob/ed651e86729477e7d2699534d63a5cace87f7878/python-package/lightgbm/basic.py#L571-L577

Reply:
oh ok, I'll remove it

Reply:
removed in https://github.com/microsoft/LightGBM/pull/3774/commits/440a30bfe3a7f18ab989c61387b44e68cf3bd1e8